src/components/policy/policy_external/src/cache_manager.cc

```diff
@@ -1067,7 +1068,8 @@ void CacheManager::ResetIgnitionCycles() {
 
 int CacheManager::TimeoutResponse() {
   CACHE_MANAGER_CHECK(0);
-  return pt_->policy_table.module_config.timeout_after_x_seconds;
+  return pt_->policy_table.module_config.timeout_after_x_seconds *
```

Comment:
@LuxoftAKutsan 
There are no tests for CacheManager and furthermore we have to remove it in nearest future so if conversion will be done in PolicyManager::TimeoutResponse - its okay, but cache has to return original value in seconds.